src/main/java/jnr/constants/platform/fake/Errno.java

```diff
@@ -12,7 +12,7 @@
 ENOEXEC(8),
 EBADF(9),
 ECHILD(10),
-EDEADLK(11),
```

Comment:
EDEADLK is what EWOULDBLOCK was before.  I see changing values in this.  Can this be right?  It is not the only case

Reply:
This appears to match your results (offline discussion). Could be ordering of generated values that changed?